internal/handlers/handlers.go

```diff
@@ -1 +1,73 @@
 package handlers
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func HtmlHandler(w http.ResponseWriter, r *http.Request) {
+
+	possiblePaths := []string{
+		"index.html",
+		"../index.html",
+	}
+	var data []byte
+	for _, path := range possiblePaths {
+		d, err := os.ReadFile(path)
+		if err == nil {
+			data = d
+			log.Printf("Успешно загружен файл из: %s", path)
+			break
+		}
+	}
+	/* file, err := os.ReadFile("../index.html")
+	if err != nil {
+		http.Error(w, "Ошибка чтения файла", http.StatusInternalServerError)
+		return
+	} */
+
+	w.Header().Set("Content-Type", "text/html;charset=utf-8")
+	w.WriteHeader(http.StatusOK)
+	w.Write(data)
+
+}
+
+func UploadHandler(w http.ResponseWriter, r *http.Request) {
+	r.ParseMultipartForm(10 << 20)
+
+	file, _, err := r.FormFile("myFile")
+	if err != nil {
+		http.Error(w, "ошибка при получении файла", http.StatusInternalServerError)
```

Comment:
Это проблема клиента
```suggestion
		http.Error(w, "ошибка при получении файла", http.StatusBadRequest)
```